ui/helpers/utils/i18n-helper.tsx

```diff
@@ -165,3 +166,13 @@ async function fetchRelativeTimeFormatData(languageTag: string): Promise<Json> {
   );
   return await response.json();
 }
+
+export function getNetworkLabelKey(network: string): string {
```

Comment:
Why do these two networks need a different "label key" than the others?

Reply:
It is because `app/_locales` `messages.json` files, use networks name as keys. `linea-goerli` and `linea-mainnet` networks have this `-` character in their name . `app/_locales` `message.json` files cannot have keys containing `-` character.
That's why I use `lineaGoerli` and `lineaMainnet` as keys in `app/_locales` files and that's why I need this `getNetworkLabelKey` function to map real network names with `app/_locales` `message.json` files keys.

Reply:
Thanks, makes sense.

This does go against a guideline we have about using variables for the `t` function - we prefer to use string literals instead so that we can more effectively find unused localized messages and remove them. But this was a pre-existing pattern in this case and you're just extending it, so this seems fine for this PR

Reply:
I agree that is not a very clean solution but it is the only solution I found. If you have another suggestion feel free to share it  😄

Reply:
Our solution elsewhere is to create a mapping of keys (e.g. network name) to localized _messages_ (not message keys), e.g. as we've done here: https://github.com/MetaMask/metamask-extension/blob/develop/shared/notifications/index.js

While I would prefer that solution, I'm actually glad that you haven't done it here because it would increase the scope of the PR too much. I'm just mentioning it here for future reference.